tests/integration/kubernetes/k8s-ro-volume.bats

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bats
+#
+# Copyright (c) 2021 Ant Group
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+load "${BATS_TEST_DIRNAME}/../../common.bash"
+load "${BATS_TEST_DIRNAME}/tests_common.sh"
+
+setup() {
+	[ "${KATA_HYPERVISOR}" == "firecracker" ] && skip "test not working see: ${fc_limitations}"
+	pod_name="test-readonly-volume"
+	container_name="busybox-ro-volume-container"
+	tmp_file="ro-volume-test-foobarfoofoo"
+	ro_volume_suffix="-tmp"
+	if exec_host "$node" which apt-get; then
```

Comment:
I was wondering where `$node` is defined. I think you need `node="$(get_one_kata_node)"` or something. I am seeing this test spawn another `custom-node-debugger` pod on top of the existing one like:

```
1..1
not ok 1 Test readonly volume for pods
# (in test file k8s-ro-volume.bats, line 32)
#   `kubectl wait --for=condition=Ready --timeout=$timeout pod "$pod_name"' failed
# error: timed out waiting for the condition on pods/custom-node-debugger-da39a3e
# Error from server (BadRequest): pod custom-node-debugger-da39a3e does not have a host assigned
# pod/node-debugger-kata-runner-01-snzwd condition met
# pod "node-debugger-kata-runner-01-snzwd" deleted
# [bats-exec-test:68] INFO: k8s configured to use runtimeclass
# pod/test-readonly-volume created
# error: timed out waiting for the condition on pods/test-readonly-volume
# pod "test-readonly-volume" deleted
# error: timed out waiting for the condition on pods/custom-node-debugger-0f4e40e
# Error from server (BadRequest): pod custom-node-debugger-0f4e40e does not have a host assigned
# Error from server (BadRequest): pod custom-node-debugger-0f4e40e does not have a host assigned
[run_kubernetes_tests.sh:125] INFO: NAMESPACE      NAME                                      READY   STATUS      RESTARTS   AGE
kube-system    custom-node-debugger-21c1c33              1/1     Running     0          34m
kube-system    custom-node-debugger-da39a3e              0/1     Pending     0          2m34s
kube-system    custom-node-debugger-0f4e40e              0/1     Pending     0          30s
```
https://github.com/kata-containers/kata-containers/actions/runs/11022039234/job/30611782441

which means a node name (in the workflow, a single node is used) is not properly passed to the function.